### DISCUSSION TITLE
Save as type

### DISCUSSION THREAD

**Discussion by lz1nwm** [JanMarvin/openxlsx2#1041] [Q&A] [answered]
Is it possible to implement saving spreadsheets as different type e.g. 'pdf'?

**Comment by JanMarvin** [comment, marked as answer]
Hi @lz1nwm , I assume that you want to save a spreadsheet in a pdf as if you're printing from spreadsheet software? No sorry, this is not possible. It might be possible to run something like libreoffice in headless mode to export a spreadsheet to pdf, but direct export from openxlsx2 to pdf is not possible. The entire rendering process of the file happens in the spreadsheet software, we simply create some xml code that is interpreted by the spreadsheet software.

There are various attempts to render spreadsheets in javascript which could be exported as pdf, but these are different projects that are unrelated to openxlsx2.

**Reply by JanMarvin** [reply]
Maybe something like this could work: https://stackoverflow.com/a/30465397/12340029

**Reply by lz1nwm** [reply]
Thanks @JanMarvin !
